themes/qgrid/layout/layout.ejs

```diff
@@ -47,32 +49,41 @@
     <!-- main page styles -->
     <%- css(isIndex ? 'css/index' : 'css/page') %>
 
-      <!-- this needs to be loaded before guide's inline scripts -->
-      <script src="<%- url_for('/js/menu.js') %>"></script>
-      <script>window.PAGE_TYPE = "<%- page.type %>"</script>
+    <!-- this needs to be loaded before guide's inline scripts -->
+    <script src="<%- url_for('/js/menu.js') %>"></script>
+    <script>window.PAGE_TYPE = "<%- page.type %>"</script>
 
-  </head>
+</head>
 
-  <body class="<%- isIndex ? 'index' : 'pages' %>">
+<body class="<%- isIndex ? 'index' : 'pages' %>">
 
-    <header>
-      <%- partial('partials/header') %>
-    </header>
+<header>
+    <%- partial('partials/header') %>
+</header>
 
-    <nav>
-      <%- partial('partials/nav') %>
-    </nav>
+<aside id="nav">
+    <div class="nav-header">
+        <button class="hamburger nav-trigger"></button>
```

Comment:
nav-trigger is not enought? do we need hamburger? )

Reply:
@klumba12 I wanted to separate js-classes from styles :-)  We can name all our js-classes with 'js-' prefix. What do you think?

Reply:
this is not js class, the name is obvious enough

Reply:
👌 🆗